ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -0,0 +1,215 @@
+    ELSE
+        Click Button    ${PIPELINES_IMPORT_BTN_FORM_XP}
+    END
+    Wait Until Generic Modal Disappears
+    Wait Until Project Is Open    project_title=${project_title}
+
+Create Pipeline Run    # robocop: disable
+    [Documentation]    Create a pipeline run from DS Project details page.
+    ...                Note that the ${type} arguments can accept:
+    ...                - immediate
+    ...                - schedule -> this requires to insert additional arguments
+    ...                TEMPORARILY SUPPORTING ONLY IMMEDIATE RUNS AND DEFAULT MODEL PARAMS
+    [Arguments]    ${name}    ${pipeline_name}    ${from_actions_menu}=${TRUE}    ${run_type}=Immediate
+    ...            ${trigger_type}=Periodic    ${start_date}=${NONE}    ${start_time}=${NONE}    ${end_date}=${NONE}
+    ...            ${end_time}=${NONE}    ${cron_expr}=${NONE}    ${press_cancel}=${FALSE}    &{model_param}
+    IF    ${from_actions_menu} == ${FALSE}
+        Show Pipeline Runs Section    pipeline_name=${pipeline_name}
+        Wait Until Page Contains Element    ${CREATE_RUN_BTN_XP}
+        Click Element    ${CREATE_RUN_BTN_XP}
+    ELSE
+        Pipelines.Click Action From Actions Menu    pipeline_name=${pipeline_name}
+        ...    action=Create run
+    END
+    Wait For RHODS Dashboard To Load    expected_page=Create run
+    ...    wait_for_cards=${FALSE}
+    Fill In Run Creation Form    name=${name}    pipeline_name=${pipeline_name}
+    ...    run_type=${run_type}    trigger_type=Periodic    start_date=${start_date}
+    ...    start_time=${start_time}    end_date=${end_date}    end_time=${end_time}
+    ...    cron_expr=${cron_expr}    &{model_param}
+    IF    ${press_cancel} == ${TRUE}
+        Click Button    ${GENERIC_CANCEL_BTN_XP}
+        Wait For RHODS Dashboard To Load    expected_page=Pipelines
+        ...    wait_for_cards=${FALSE}
+        ${workflow_name}=    Set Variable    ${NONE}
+    ELSE
+        Click Element    ${GENERIC_CREATE_BTN_XP}
+        Wait Until Page Contains Run Topology Page    run_name=${name}
+        ${workflow_name}=    Get Workflow Name From Topology Page
+    END
+    RETURN    ${workflow_name}
+
+Click Action From Actions Menu
+    [Documentation]    Click an action from Actions menu (3-dots menu on the right)
+    [Arguments]    ${pipeline_name}    ${action}
+    Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${pipeline_name}"]]/td[contains(@class,"pf-c-table__action")]/div/button[@aria-label="Actions"]    # robocop: disable
+    Wait Until Page Contains Element       xpath=//tr[td[@data-label="Name"]//*[text()="${pipeline_name}"]]/td[contains(@class,"pf-c-table__action")]/div/ul/li/button[text()="${action}"]    # robocop: disable
+    Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${pipeline_name}"]]/td[contains(@class,"pf-c-table__action")]/div/ul/li/button[text()="${action}"]    # robocop: disable
+
+Pipeline Should Be Listed
```

Comment:
> it won't fail in that case.
it will fail. It will just do not stop the execution of the test. This is to reduce the possibility to block a test execution in case this kw fails (e.g., timing issue, or xpath changes etc).